### PR TITLE
Use function definitions

### PR DESCRIPTION
**Pull request by bethanyaconnor**
_Finally_ getting function descriptions translated.

This PR is a followup to #34568 and #34735. We can do this now that the down/out sync from this week is merged (#34857).

## Links

- [spec]()
- [jira]()

## Testing story

# Reviewer Checklist:

- [ ] Tests provide adequate coverage
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: dashboard/test/helpers/levels_helper_test.rb [hunk 1]
```diff
@@ -651,17 +651,23 @@ def stub_country(code)
     toolbox_translated_name = "Azioni"
     @level.toolbox_blocks = toolbox
 
-    start = "<xml><block type=\"procedures_defnoreturn\"><title name=\"NAME\">details</title></block></xml>"
+    start = "<xml><block type=\"procedures_defnoreturn\"><mutation><description>function description</description></mutation><title name=\"NAME\">details</title></block></xml>"
```
Comment (Hamms) [inline dashboard/test/helpers/levels_helper_test.rb:654]: I'd love to see a test that works with either just name or name+description, to confirm that the minimal case still works

**Review by Hamms — APPROVED** [review]
one minor testing nit, otherwise this looks great!

File: dashboard/test/helpers/levels_helper_test.rb (hunk 1 above)
Comment (bethanyaconnor) [reply dashboard/test/helpers/levels_helper_test.rb:654]: This one works for name+description and there is a different test in this file that only translates the name (though there's not description in that block). I can add more tests that test different combinations, including fallback.

Comment (Hamms) [reply dashboard/test/helpers/levels_helper_test.rb:654]: oh, sorry, to be more clear: I want to see a test that operates over a block that only _defines_ the name and not the description. Just to make sure the i18n logic doesn't panic when given minimal XML

Comment (Hamms) [reply dashboard/test/helpers/levels_helper_test.rb:654]: looking at the other test you mention, I think it does just want I want!

**Pull request merged**